subscriptions/sales_flow/renewals.rst

```diff
@@ -0,0 +1,82 @@
+====================
+Renew a subscription
+====================
+
+The key feature of a subscription business model is the recurring nature of payments. In this model,
+customers pay a recurring amount in exchange for access to a product or a service.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>The renewal of a
+   subscription is the process followed by each customer when willing to pursue a
+   subscription.</b> </div>
+
+Each subscriber experiences this renewal process monthly, annually or sometimes more, depending on
+the duration of the contract. Most subscription companies choose to automate their renewal
+processes, but in some cases, manual subscription renewals are still the preferred option.
+
+With **Odoo Subscriptions**, you can have all your subscriptions in one application, suggest to your
+customers an automatic subscription renewals as a manual one and finally, filter all your
+subscriptions and easily find those to renew (with the help of the tag *To renew*).
+
+Renew your first subscription
+=============================
+
+Before renewing a subscription, be sure to check out our documentation on how to create a quotation
+using subscription products, :doc:`../../subscriptions/sales_flow/create_a_quotation`. Indeed,
+once confirmed, a quotation becomes a sales order and, a new subscription is automatically created.
+This subscription will therefore have the status *In progress*. From there, you have the
+possibility to renew the subscription. In the Other Info tab, underneath the To Renew section, you
+can activate the *To renew* option. When activated, a yellow tag will automatically appear in the
+upper right corner of the subscription.
```

Comment:
```suggestion
Therefore, this subscription has the *In progress* status. From there, you have the
possibility to renew the subscription. In the Other Info tab, underneath the *To Renew* section, you
can activate the *To renew* option. When activated, a yellow tag automatically appears in the
upper right corner of the subscription.
```

Reply:
Ok for all changes. Except *To renew* section because it is already in italic with the option "To renew" :) Section / Option. I prefer italic for features, not for section :)